tests/integration/test_kinesis.py

```diff
@@ -23,6 +23,7 @@ def assert_consumers(count):
 
         # create stream and assert 0 consumers
         client.create_stream(StreamName=stream_name, ShardCount=1)
+        sleep(1)
```

Comment:
kinesis-mock wont be able to delete a stream that is currently CREATING so added a sleep here.